### PR TITLE
fix docs

### PR DESCRIPTION
**Pull request by ndalliard**
changelog-none

### CODE REVIEW

File: doc/lightning-cli.1 [hunk 1]
```diff
@@ -127,7 +127,7 @@ pretty printing of results isn’t pretty\.
 
 .SH AUTHOR
 
-Rusty Russell \fI<rusty@rustcorp.com.au\fR> is mainly to blame\.
+Rusty Russell <\fIrusty@rustcorp.com.au\fR> is mainly to blame\.
```
Comment (vincenzopalazzo) [inline doc/lightning-cli.1:130]: mh the file without `.md` should be autogenerated from the `make doc-all` command, and here I'm missing the change in the `md `file

**Review by vincenzopalazzo — COMMENTED** [review]
LGTM.

The only nit is the message as a comment, I'm missing the change the the `md` file.

P.S: The failure in the github action is not related to this PR.

**Comment by ndalliard** [comment]
hmm, i just realised from your nit, that the manpage files are autogenerated in a wrong way

should be:
````man
<\fIrusty@rustcorp.com.au\fR>
````
generated:
````man
\fI<rusty@rustcorp.com.au\fR>
````

i will just squash the commits into one commit with the other fixes and leave the above as it is, it probably has something to do with how `mrkd` generates the manpages, but i am not familiar with it

**Comment by vincenzopalazzo** [comment]
>i will just squash the commits into one commit with the other fixes and leave the above as it is, it probably has something to do with how mrkd generates the manpages, but i am not familiar with it

ack!

Yes, the pages are autogenerated by the python module `mrkd`.

I think if you swash your change in a unique commit it is good too

**Comment by ndalliard** [comment]
are you okay with me adding the generated files into the `.gitignore`?

**Comment by vincenzopalazzo** [comment]
>are you okay with me adding the generated files into the .gitignore?

I think is not a good choice because the command `make` doesn't call the `make doc-all`.

**Comment by ndalliard** [comment]
> > are you okay with me adding the generated files into the .gitignore?
> 
> I think is not a good choice because the command `make` doesn't call the `make doc-all`.

okay, the `gmake doc-all` now generated a lot of empty lines, leave them like this or revert to how it was?

**Comment by vincenzopalazzo** [comment]
>okay, the gmake doc-all now generated a lot of empty lines, leave them like this or revert to how it was?

I would like to have the previous status with your typo fix in the` lightning-config` (that is my fault when I made the doc page)

**Comment by ndalliard** [comment]
updated - kept the other changes, the empty lines are strange, but if i remove them i guess the SHA256 wouldn't be correct

**Review by vincenzopalazzo — APPROVED** [review]
ack https://github.com/ElementsProject/lightning/pull/4622/commits/f7bd80f6a5d4242b1a4f78618e9621fb5bbb02b4

**Comment by rustyrussell** [comment]
We check in the generated files so that you don't need mrkd to build them, which is why they're not .gitignore'd.

Thanks for finding this!

**Pull request merged**